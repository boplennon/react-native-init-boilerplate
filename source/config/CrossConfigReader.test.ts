import ConfigReader from './CrossConfigReader';

describe('ConfigReader', () => {
  it('Should match snapshot', () => {
    expect(ConfigReader).toMatchSnapshot();
  });

  it('`GetEnv()` should return instance', () => {
    const config = ConfigReader.GetEnv();
    expect(config).toBeDefined();
  });

  it('Env config should match snapshot', () => {
    const config = ConfigReader.GetEnv();
    expect(config).toMatchSnapshot();
  });

  it('Config should have APP_NAME', () => {
    const config = ConfigReader.GetEnv();
    expect(config.APP_NAME).toBeDefined();
  });
});
