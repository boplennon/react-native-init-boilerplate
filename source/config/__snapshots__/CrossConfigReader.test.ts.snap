// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfigReader Env config should match snapshot 1`] = `
Object {
  "API_URL": "https://api.github.com/graphql",
  "APP_NAME": "MyApp",
  "APP_PREFIX": "myapp://",
  "GOOGLE_MAPS_API_KEY": "abcdefgh",
  "SENTRY_URI": "https://6bedde4a9c0b4bc5b5a2b17973edc460@sentry.io/1284889",
}
`;

exports[`ConfigReader Should match snapshot 1`] = `
Object {
  "GetEnv": [Function],
}
`;
